### PR TITLE
Default Intro

### LINKED ISSUE #42: Default Intro in iFrame
## 🚀 Feature

Default Intro in the iframe, which disappears just as we start typing.

## Have you spent some time to check if this issue has been raised before?

Yes

### Have you read the [Contributing Guidelines](https://github.com/s-katte/React-Code-Pen/blob/master/CONTRIBUTING.md)?

Yes

## Pitch

This improves its look as the present one is plain.
We can have there some glimpse of features of Sketchify.

### PR DESCRIPTION
**Pull request by im-adithya**
## https://github.com/s-katte/Sketchify/issues/42

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

## Description of your PR

Provides default intro in the iFrame

## Screenshots (if applicable)

(Add screenshots of the visual changes in the project)

## Test Plan

I added another iFrame box with a display-block class and on interchanging the class with the previous iFrame, it worked as it previously did.

### Have you read the [Contributing Guidelines](https://github.com/s-katte/React-Code-Pen/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -57,8 +79,14 @@ function App() {
             document.title = "Sketchify - Untitled";
         } else {
             document.title = "Sketchify - " + title;
-            localStorage.setItem("codepen-clone-title", title);
+            localStorage.setItem("sketchify-title", title);
```
Comment (s-katte) [inline src/components/App.js]: Please change the spelling here to keep the code consistent.

**Review by s-katte — CHANGES_REQUESTED** [review]
Also, next time please follow the CONTRIBUTING guideline mentioned to avoid extra commits and irregularity in the flow.

File: src/components/App.js (hunk 1 above)
Comment (im-adithya) [reply src/components/App.js]: Sorry I didn't get what you are trying to say...Did you mean capitalizing sketchify?

Comment (s-katte) [reply src/components/App.js]: Yes

Comment (im-adithya) [reply src/components/App.js]: @s-katte The local storage keys are all set with the "sketchify-title-" prefix; that's why I did this.
Should I still make changes?

Comment (s-katte) [reply src/components/App.js]: okay yes, so sorry.

**Review by s-katte — APPROVED** [review]
Everything else looks good, merging it.

**Review by s-katte — CHANGES_REQUESTED** [review]
After some testing, I constantly found that there is an error in the code.
Steps to reproduce error:
- Add a title
- Refresh the page

You can see the error in the console of browser.

Please solve that error, as this occured in only this PR, you can use https://deploy-preview-48--sketchify.netlify.app/

**Comment by im-adithya** [comment]
The deploy preview is empty but it's working in my local fork... I don't know what to do now!
Please help @s-katte

**Comment by s-katte** [comment]
> The deploy preview is empty but it's working in my local fork... I don't know what to do now!
> Please help @s-katte

Okay, I will merge and try if same issue persists on dev branch.

**Review by s-katte — APPROVED** [review]

**Comment by im-adithya** [comment]
@s-katte All of my contributions are not counting towards hacktoberfest... It is now showing this as an ineligible repo!

**Comment by s-katte** [comment]
> @s-katte All of my contributions are not counting towards hacktoberfest... It is now showing this as an ineligible repo!

were your PRs matured? Tell me if it is now eligible, I have added label now.

**Comment by im-adithya** [comment]
No @s-katte all my PRs were about to mature in 2-3 days, nothing got accepted as of now. Also, there's a spelling mistake in repo's Hacktoberfest label. I think it is because of that.

**Comment by s-katte** [comment]
> No @s-katte all my PRs were about to mature in 2-3 days, nothing got accepted as of now. Also, there's a spelling mistake in repo's Hacktoberfest label. I think it is because of that.

Thanks for correction, updated that.

**Comment by im-adithya** [comment]
@s-katte It is still showing ineligible. Can you remove and add the hacktoberfest-accepted label now (after updating the repo label) to my other PRs as well?
I have done 3 PRs. None of them got matured now. This is so unfair!

**Comment by s-katte** [comment]
> @s-katte It is still showing ineligible. Can you remove and add the hacktoberfest-accepted label now (after updating the repo label) to my other PRs as well?
> I have done 3 PRs. None of them got matured now. This is so unfair!

Yes will do it

**Comment by im-adithya** [comment]
It's not updated yet @s-katte. And I don't think it will now. Btw, thanks for your immediate response!
Learned a lot from the project anyway!! 🍻

**Comment by s-katte** [comment]
> It's not updated yet @s-katte. And I don't think it will now. Btw, thanks for your immediate response!
> Learned a lot from the project anyway!!

I am glad to have you as a contributor. Looking forward to having you as an active contributor here.

**Pull request merged**